tests/conftest.py

```diff
@@ -251,3 +262,23 @@ def fixture_tower_task() -> TowerTask:
         file_format=FileFormat.JSON, file_path=TowerTaskResponseFile.RUNNING
     )
     return TowerTask(**tower_task_running_content["tasks"][0]["task"])
+
+
+@pytest.fixture(scope="session")
+def slurm_squeue_output(squeue_dir: Path) -> Dict[str, Path]:
+    """Return SLURM squeue output for analysis started via SLURM."""
+    file_postfix: str = f"squeue{FileExtension.CSV}"
+    return {
+        "blazinginsect": Path(squeue_dir, f"blazinginsect_{file_postfix}").as_posix(),
+        "crackpanda": Path(squeue_dir, f"crackpanda_{file_postfix}").as_posix(),
+        "cuddlyhen": Path(squeue_dir, f"cuddlyhen_{file_postfix}").as_posix(),
+        "daringpidgeon": Path(squeue_dir, f"daringpidgeon_{file_postfix}").as_posix(),
+        "escapedgoat": Path(squeue_dir, f"escapegoat_{file_postfix}").as_posix(),
+        "fancymole": Path(squeue_dir, f"fancymole_{file_postfix}").as_posix(),
+        "happycow": Path(squeue_dir, f"happycow_{file_postfix}").as_posix(),
+        "lateraligator": Path(squeue_dir, f"lateraligator_{file_postfix}").as_posix(),
+        "liberatedunicorn": Path(squeue_dir, f"liberatedunicorn_{file_postfix}").as_posix(),
+        "nicemice": Path(squeue_dir, f"nicemice_{file_postfix}").as_posix(),
+        "rarekitten": Path(squeue_dir, f"rarekitten_{file_postfix}").as_posix(),
+        "trueferret": Path(squeue_dir, f"trueferret_{file_postfix}").as_posix(),
+    }
```

Comment:
Perhaps a bit easier on the eyes.
```suggestion
@pytest.fixture(scope="session")
def slurm_squeue_output(squeue_dir: Path) -> Dict[str, Path]:
    """Return SLURM squeue output for analysis started via SLURM."""
    file_postfix: str = f"squeue{FileExtension.CSV}"
    
    job_names = [
        "blazinginsect", "crackpanda", "cuddlyhen", "daringpidgeon", "escapedgoat",
        "fancymole", "happycow", "lateraligator", "liberatedunicorn", "nicemice",
        "rarekitten", "trueferret"
    ]
    
    squeue_output = {}
    
    for job_name in job_names:
        squeue_output[job_name] = Path(squeue_dir, f"{job_name}_{file_postfix}").as_posix()
    
    return squeue_output
```